### PR TITLE
clip 0.7 (new formula)

### PR DESCRIPTION
**Pull request by ankane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Command line charts 🎉

### CODE REVIEW

File: Formula/clip.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "install"
+    pkgshare.install "examples", "tests"
+  end
+
+  test do
+    cp_r pkgshare/"examples", testpath
+    cp_r pkgshare/"tests", testpath
```
Comment (SMillerDev) [inline Formula/clip.rb:24]: You're not using these right?

Comment (ankane) [reply Formula/clip.rb:24]: The examples reference data from `tests`.

https://github.com/asmuth/clip/blob/v0.5/examples/charts-basic/areachart.clp#L6-L7

**Review by SMillerDev — APPROVED** [review]
Thanks @ankane ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by ankane** [comment]
Looks like there was a build failing on High Sierra. Will follow up with the project.

**Review by SMillerDev — CHANGES_REQUESTED** [review]
Build failing

**Comment by ankane** [comment]
Looks like we're good now.

**Review by SMillerDev — APPROVED** [review]

**Comment by ankane** [comment]
Thanks @SMillerDev!

**Pull request closed without merging**